File: Net_Framework/VB/GrowthWare.Framework/Model/Profiles/MRequestNewPassword.vb
Namespace Model.Profiles
    ''' <summary>
    ''' Model object representing the Request New Password Profile
    ''' </summary>
    <Serializable(), CLSCompliant(True)> _
    Public Class MRequestNewPassword
        Inherits MMessageProfile

        Public Sub New()

        End Sub

        Public Property AccountName As String

        Public Property FullName As String

        Public Property Password As String

        Public Property Server As String

        Public Sub New(ByVal profile As MMessageProfile)
            MyBase.New(profile)
        End Sub
    End Class
End Namespace

File: Net_Core/Framework/Models/MRequestNewPassword.cs
using System;

namespace GrowthWare.Framework.Models
{
    /// <summary>
    /// Model object representing the Request New Password Profile
    /// </summary>
    [Serializable(), CLSCompliant(true)]
    public class MRequestNewPassword : MMessage
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="MRequestNewPassword"/> class.
        /// </summary>
        public MRequestNewPassword() : base() { }

        /// <summary>
        /// Initializes a new instance of the <see cref="MRequestNewPassword" /> class.
        /// </summary>
        /// <param name="profile">MMessage</param>
        public MRequestNewPassword(MMessage profile)
            : base(profile)
        {

        }

        /// <summary>
        /// Gets or sets the name of the account.
        /// </summary>
        /// <value>The name of the account.</value>
        public String AccountName { get; set; }

        /// <summary>
        /// Gets or sets the full name.
        /// </summary>
        /// <value>The full name.</value>
        public String FullName { get; set; }

        /// <summary>
        /// Gets or sets the password.
        /// </summary>
        /// <value>The password.</value>
        public String Password { get; set; }

        /// <summary>
        /// Gets or sets the server.
        /// </summary>
        /// <value>The server.</value>
        public String Server { get; set; }

        /// <summary>
        /// Gets or sets the token.
        /// </summary>
        public String ResetToken { get; set; }
    }
}
